src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <title>ASP.NET Core Logging Integration</title>
     <Description>This site extension enables logging integration for ASP.NET Core applications on Azure App Service.</Description>
-    <TargetFramework>$(DefaultNetFxTargetFramework)</TargetFramework>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework);netstandard2.0</TargetFrameworks>
```

Comment:
Does it make sense to cross-target site extensions? I thought these were built for specific target frameworks in a separate build. I could be wrong though so it's worth double checking. Same applies to LoggingBranch.

Reply:
We only build site extensions on Windows but that doesn't make this useless. @jkotalik or @BrennanConroy would `net6.0` site extension assemblies be useful❔

Reply:
Isn't the point of cross-targeting to get nicer behavior when consuming the package(s) (and sometimes perf benefits if you use newer APIs for different targets)?

Most use cases of the site extension are through the Azure Portal and so wouldn't care about the behavior consuming this package.

Reply:
Reverted this change